pkg/pillar/scripts/device-steps.sh

```diff
@@ -395,6 +395,20 @@ if [ ! -s $CONFIGDIR/device.cert.pem ] || [ $RTC = 0 ]; then
         sleep 10
         YEAR=$(date +%Y)
     done
+    # Update RTC based on time from ntpd so that after a reboot we have a sane
+    # starting time. This fixes issues when the RTC was not in UTC
+    hwclock -u -v --systohc
```

Comment:
It seems we should run it only in case of `$RTC = 1`

Reply:
Makes sense to add that check.